classes/actions/ShoppingfeedOrderImportActions.php

```diff
@@ -1401,4 +1404,30 @@ protected function addJustEnoughStock($id_product, $id_product_attribute, $refer
             $this->getIdShop()
         );
     }
+
+    protected function addDniToAddress(Address $address, OrderResource $apiOrder)
```

Comment:
Shouldn’t be better in a import rules ? Because additionalFields.buyer_identification_number is probably a specific implementation of marketplace